built-in-services/account/src/types.rs

```diff
@@ -0,0 +1,123 @@
+use serde::{Deserialize, Serialize};
+
+use bytes::Bytes;
+use protocol::fixed_codec::{FixedCodec, FixedCodecError};
+use protocol::types::{Address, Hash, Hex};
+use protocol::ProtocolResult;
+
+pub const ACCOUNT_TYPE_PUBLIC_KEY: u8 = 0;
+pub const ACCOUNT_TYPE_MULTI_SIG: u8 = 1;
+pub const MAX_PERMISSION_ACCOUNTS: u8 = 16;
+
+#[derive(Deserialize, Serialize, Clone, Debug)]
+pub struct VerifyPayload {
+    pub tx_hash: Hash,
+    pub witness: Bytes,
+}
+
+#[derive(Deserialize, Serialize, Clone, Debug, Default)]
+pub struct VerifyResponse {
+    pub address: Address,
+}
+
+#[derive(Deserialize, Serialize, Clone, Debug)]
+pub struct GetAccountPayload {
+    pub user: Address,
+}
+
+#[derive(Deserialize, Serialize, Clone, Debug)]
+pub struct GenerateAccountPayload {
+    pub accounts:  Vec<PayloadAccount>,
+    pub threshold: u8,
+}
+
+#[derive(Deserialize, Serialize, Clone, Debug, Default)]
+pub struct GenerateAccountResponse {
+    pub accounts:  Vec<PayloadAccount>,
+    pub threshold: u8,
```

Comment:
`threshold` and `accounts` are useless in response.

Reply:
Does not outside client or muta browser  need this?

Reply:
We use `event` in this case. If we invoke `generate_account` method from other service or contract, we can only know the change from event. If we do need them, we can get them from transaction payload.

Reply:
ok